### PR TITLE
MT3-659 #ready-for-test Add missing information to Review page

### PR DESCRIPTION
**Pull request by Gweaton**
- Tenancy Start date
- Helper text at bottom of page

### CODE REVIEW

File: pages/thc/[processRef]/review.tsx [hunk 1]
```diff
@@ -121,25 +129,18 @@ const ReviewPage: NextPage = () => {
...
-                allTenantNames.length > 0
-                  ? allTenantNames.join(", ")
-                  : "Loading...",
-            },
-            {
-              key: "Tenure type",
-              value: tenureType.result ? tenureType.result : "Loading...",
-            },
-            { key: "Ouside Property:", value: outsidePropertyImageThumbnails },
+            { key: "Outside Property:", value: outsidePropertyImageThumbnails },
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:143]:
```suggestion
            { key: "Outside Property", value: outsidePropertyImageThumbnails },
```

I think (without looking).

File: pages/thc/[processRef]/review.tsx [hunk 2]
```diff
@@ -121,25 +129,18 @@ const ReviewPage: NextPage = () => {
...
     >
       <React.Fragment>
+        <TenancySummary
+          details={{
+            address: address.result,
+            tenants: allTenantNames,
+            tenureType: tenureType.result,
+            startDate: tenancyStartDate.result,
+          }}
+        />
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:139]: Does it make semantic sense to have two lists? I like that we reuse `TenancySummary`, but it doesn't feel like it's correct to have two lists.

Maybe we could add an `extraRows` prop to `TenancySummary`?

File: pages/thc/[processRef]/review.tsx (hunk 1 above)
Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx:143]: I actually took this off the balsamiq design, but can get rid of it if you think it'd look better without?

File: pages/thc/[processRef]/review.tsx (hunk 2 above)
Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx:139]: Yeah, might be a better idea

File: pages/thc/[processRef]/review.tsx (hunk 1 above)
Comment (erbridge) [reply pages/thc/[processRef]/review.tsx:143]: I think it's more consistent?

File: pages/thc/[processRef]/review.tsx (hunk 2 above)
Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx]: I've given this a go, but I'm not 100% convinced about my implementation - if you could take a look and let me know if I've got the wrong end of the stick, that would be great!

File: components/TenancySummary.tsx [hunk 3]
```diff
@@ -11,40 +10,50 @@ interface Props {
...
-              : loading,
-          },
-        ]}
+        rows={
+          extraRows
+            ? [
+                ...rows,
+                ...((extraRows as unknown) as { key: string; value: string }[]),
+              ]
+            : rows
```
Comment (erbridge) [inline components/TenancySummary.tsx:55]: Should be able to write this like `[...rows, ...(((extraRows || []) as unknown) as { key: string; value: string }[])]`, to be flatter.

File: pages/thc/[processRef]/review.tsx [hunk 4]
```diff
@@ -121,25 +124,15 @@ const ReviewPage: NextPage = () => {
...
-            { key: "Ouside Property:", value: outsidePropertyImageThumbnails },
+        <TenancySummary
+          details={{
+            address: address.result,
+            tenants: allTenantNames,
+            tenureType: tenureType.result,
+            startDate: tenancyStartDate.result,
+          }}
+          extraRows={[
+            { key: "Outside Property", value: outsidePropertyImageThumbnails },
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:135]:
```suggestion
            { key: "Outside property", value: outsidePropertyImageThumbnails },
```

**Review by erbridge — APPROVED** [review]
Looks sensible to me. Some minor comments, but not really a blocker.

**Review by erbridge — APPROVED** [review]

**Pull request merged**